webapp/apps/taxbrain/models.py

```diff
@@ -768,25 +768,24 @@ class TaxSaveInputs(models.Model):
     # Creation DateTime
     creation_date = models.DateTimeField(default=datetime.datetime(2015, 1, 1))
 
-
-    @property
-    def tax_result(self):
+    def get_tax_result(self):
```

Comment:
This seems like a good idea - I think the best "logic" for deciding if something should be a property is if:

1. It is computationally light, or
2. It is cached (i.e. the result is only computed once)

Reply:
Ok, that makes sense. Thanks for the info. I also couldn't get the property approach to work without deleting the column in the database.